include/ck/tensor_operation/gpu/device/impl/device_grouped_gemm_multiple_d_dl.hpp

```diff
@@ -607,6 +607,9 @@ struct DeviceGroupedGemmMultipleD_Dl : public DeviceGroupedGemm<ALayout,
                 }
             }
 
+            // If user provides copy stream and copy event, we assume that he's also
```

Comment:
```suggestion
            // If the user provides copy stream and copy event, we assume that they're also
```

When referring to a user, singular "they" is used instead of "he" or "he/she".

Reply:
Changed the references to the user from "he/she" to "they" in 3 places, should be okay now